next-tavla/app/(admin)/boards/components/Column/Delete.tsx

```diff
@@ -68,7 +75,7 @@ function DeleteButton({
     type,
     onClick,
 }: {
-    type?: 'button' | 'icon'
+    type?: 'button' | 'icon' | 'action'
     onClick: () => void
 }) {
     if (type === 'button') {
```

Comment:
Trenger vi å ha med for `type == button` i det hele tatt nå? Siden vi bytter om fra button til menuItem, og button brukes ikke noen andre steder

Reply:
På liten skjerm brukes button